backend/src/Edna/ExperimentReader/Error.hs

```diff
@@ -0,0 +1,27 @@
+module Edna.ExperimentReader.Error
+  ( ExperimentParsingError (..)
+  ) where
+
+import Universum
+
+import Codec.Xlsx (ParseError)
+import Fmt (Buildable(..), (+|), (|+))
+
+data ExperimentParsingError
+  = UnexpectedCellType
+  | InvalidCell (Int, Int)
+  | EmptyCell
```

Comment:
Why do we attach cell location in `InvalidCell`, but do not attach it in `EmptyCell` and `UnexpectedCellType`?

Reply:
Note that `EmptyCell` still doesn't include `PointYX` for unclear reason.